lnbits/core/views/api.py

```diff
@@ -69,8 +69,11 @@ async def api_create_account(data: CreateWallet) -> Wallet:
             status_code=HTTPStatus.FORBIDDEN,
             detail="Account creation is disabled.",
         )
-    account = await create_user_account()
-    return await create_wallet(user_id=account.id, wallet_name=data.name)
+    account = await create_user_account(wallet_name=data.name)
+    wallets = await get_wallets(account.id)
+    assert len(wallets) != 0, "Wallet not created for user."
```

Comment:
```suggestion
    assert len(wallets) == 0, "Wallet not created for user."
```

Reply:
hm, i dont really like the assert here, i think it should be assert inside crud.

Reply:
it is